ooiservices/app/uframe/asset_tools.py

```diff
@@ -109,11 +98,12 @@ def get_assets_payload():
         # Get asset_list
         data, _ = new_compile_assets(result, compile_all=True)
         print '\n-- Completed loading assets....'
-        if not data:
+        if not data or data is None:
             message = 'Unable to process uframe assets; error creating assets_list.'
             raise Exception(message)
 
         # Cache 'asset_list'.
+        cache.delete('asset_list')
```

Comment:
Moving the delete down here from lines 96/98 and then setting the cache on the next line is the "Hot Swap" methodology we had discussed.  right @ednad ?